app/containers/Questions/Content/Content.jsx

```diff
@@ -232,47 +263,66 @@ export const Content = ({
   isModerator,
   profileInfo,
   isHomePage = false,
+  query,
   isSearchPage,
   isFeed,
   isCommunityFeed,
-}) => (
-  <div className="position-relative">
-    {/* {promotedQuestionsList && */}
-    {/*  promotedQuestionsList.map((item, index) => ( */}
-    {/*    <QuestionItem */}
-    {/*      {...item} */}
-    {/*      index={index} */}
-    {/*      first={index === 0} */}
-    {/*      last={index === questionsList.length - 1} */}
-    {/*      locale={locale} */}
-    {/*      communities={communities} */}
-    {/*      key={item.id} */}
-    {/*      isModerator={isModerator} */}
-    {/*      profileInfo={profileInfo} */}
-    {/*      isPromoted */}
-    {/*      isHomePage={isHomePage} */}
-    {/*    /> */}
-    {/*  ))} */}
-    {questionsList.map((item, index) => (
-      <QuestionItem
-        {...item}
-        isGeneral={isGeneralQuestion(item)}
-        index={index}
-        first={index === 0}
-        last={index === questionsList.length - 1}
-        locale={locale}
-        communities={communities}
-        key={item.id}
-        isModerator={isModerator}
-        profileInfo={profileInfo}
-        isHomePage={isHomePage}
-        isSearchPage={isSearchPage}
-        isFeed={isFeed}
-        isCommunityFeed={isCommunityFeed}
-      />
-    ))}
-  </div>
-);
+}) => {
+  const makeHighlighted = useCallback(
+    str => {
+      return <Highlight filter={query} str={str} />;
+    },
+    [query],
+  );
+  return (
+    <div className="position-relative">
+      {/* {promotedQuestionsList && */}
+      {/*  promotedQuestionsList.map((item, index) => ( */}
+      {/*    <QuestionItem */}
+      {/*      {...item} */}
+      {/*      index={index} */}
+      {/*      first={index === 0} */}
+      {/*      last={index === questionsList.length - 1} */}
+      {/*      locale={locale} */}
+      {/*      communities={communities} */}
+      {/*      key={item.id} */}
+      {/*      isModerator={isModerator} */}
+      {/*      profileInfo={profileInfo} */}
+      {/*      isPromoted */}
+      {/*      isHomePage={isHomePage} */}
+      {/*    /> */}
+      {/*  ))} */}
+      {questionsList.map(
+        (item, index) => {
+          const prepearedItem = {
+            ...item,
+            title: makeHighlighted(item.title),
+            content: makeHighlighted(item.content),
```

Comment:
why pass content props? it does not participate in rendering

Reply:
Done